rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/model/EjbSessionBeanModel.java

```diff
@@ -80,11 +80,11 @@
      * Contains the jndi location for this resource.
      */
     @Adjacency(label = JNDIResourceModel.TYPE, direction = Direction.OUT)
-    public JNDIResourceModel getJndiReference();
+    public Iterable<JNDIResourceModel> getJndiReferences();
 
     /**
      * Contains the jndi location for this resource.
      */
     @Adjacency(label = JNDIResourceModel.TYPE, direction = Direction.OUT)
-    public void setJndiReference(JNDIResourceModel jndiReference);
+    public void addJndiReference(JNDIResourceModel jndiReference);
```

Comment:
I had put a comment to change the setJndiReference to addJndiReference, but after thinking some more about it, there are three types of JNDI references that an EJB could have: Global, Module, App (for portable EJB standard), and legacy would have some EJB as well.  By setting this as a collection, there isn't a way to assign the JNDI to one of the specific types;  I would recommend having: 

setGlobalJndiReference
setModuleJndiReference
setAppJndiReference

etc.